test/e2e/e2e_test.go

```diff
@@ -38,7 +38,7 @@ var _ = Describe("controller", Ordered, func() {
 		Expect(utils.InstallCertManager()).To(Succeed())
 
 		By("creating manager namespace")
-		cmd := exec.Command("kubectl", "create", "ns", namespace)
+		cmd := exec.Command("kubectl", "create", "ns", namespace) //nolint:gosec
```

Comment:
@nb-ohad for example this place

Reply:
Yes, but this is test code, not code that will run as part of a csi deployment. If all of the cases are like that then we should not diable the rule globally just locally. I believe the last push did exactly that.